README.md

```diff
@@ -91,55 +89,46 @@ The simplest way to integrate authentication in your app is with OIDC through a
 
 ```kotlin
 import android.content.Context
-import com.okta.authfoundation.AuthFoundationDefaults
-import com.okta.authfoundation.client.OidcClient
-import com.okta.authfoundation.client.OidcClientResult
+import com.okta.authfoundation.AuthFoundation
 import com.okta.authfoundation.client.OidcConfiguration
-import com.okta.authfoundation.credential.CredentialDataSource.Companion.createCredentialDataSource
-import com.okta.authfoundationbootstrap.CredentialBootstrap
-import okhttp3.HttpUrl.Companion.toHttpUrl
 
 val context: Context = TODO("Supplied by the developer.")
-val oidcConfiguration = OidcConfiguration(
-    clientId = "{clientId}",
-    defaultScope = "openid email profile offline_access",
-)
-val client = OidcClient.createFromDiscoveryUrl(
-    oidcConfiguration,
-    "https://{yourOktaOrg}.okta.com/oauth2/default/.well-known/openid-configuration".toHttpUrl(),
+AuthFoundation.initializeAndroidContext(context)
+OidcConfiguration.default = OidcConfiguration(
+  clientId = "{clientId}",
+  defaultScope = "openid email profile offline_access",
+  issuer = "https://{yourOktaOrg}.okta.com/oauth2/default"
 )
-CredentialBootstrap.initialize(client.createCredentialDataSource(context))
 ```
 
 #### Create a Web Authentication Client
 
-We will create a `WebAuthenticationClient` and use it to perform authentication.
+We will create a `WebAuthentication` and use it to perform authentication.
 
 This launches a [Chrome Custom Tab](https://developer.chrome.com/docs/android/custom-tabs/) to display the login form, and once complete, redirects back to the application.
 
 ```kotlin
 import android.content.Context
 import com.okta.authfoundation.credential.Credential
-import com.okta.authfoundationbootstrap.CredentialBootstrap
 import com.okta.oauth2.AuthorizationCodeFlow
-import com.okta.webauthenticationui.WebAuthenticationClient
-import com.okta.webauthenticationui.WebAuthenticationClient.Companion.createWebAuthenticationClient
+import com.okta.webauthenticationui.WebAuthentication
 
 val context: Context = TODO("Supplied by the developer.")
 val credential: Credential
 val redirectUrl: String = TODO("signInRedirectUri supplied by the developer.")
-val webAuthenticationClient = CredentialBootstrap.oidcClient.createWebAuthenticationClient()
-when (val result = webAuthenticationClient.login(context, redirectUrl)) {
+val webAuthentication = WebAuthentication()
+when (val result = webAuthentication.login(context, redirectUrl)) {
     is OidcClientResult.Error -> {
         // Timber.e(result.exception, "Failed to login.")
         // TODO: Display an error to the user.
     }
     is OidcClientResult.Success -> {
-      credential = CredentialBootstrap.setDefaultCredential(token = result.result)
+      credential = Credential.store(token = result.result)
+      Credential.setDefaultCredential(credential)
```

Comment:
This wasn't doable because Credential.getDefaultCredential had to be a `suspend fun` (async) due to accessing `TokenStorage`, and optionally accessing biometrics. In kotlin, it's not possible to have `suspend` properties, and that's why I was forced to make getters/setters in this case.

Reply:
🤔 Why is TokenStorage async?

In Swift and Javascript, token storage is synchronous. It needs to be synchronous because this is one of the primary workflows for interacting with users, and if every function that retrieves a user credential needs to be async, it will hugely complicate the UI.

Reply:
Hmm that's really too bad :-\ Lets chat offline...